### PR TITLE
Update machinekit-developing.asciidoc

### PR DESCRIPTION
**Pull request by the-snowwhite**
Fix creation of .deb packages and copying out into repo folder ...

### CODE REVIEW

**Comment by cerna** [comment]
@the-snowwhite,
sorry for the delay. I wrote this two days ago, but haven't clicked the confirmation button until now (as I didn't know I have to).

**Tell me when you want to merge this. **

File: docs/developing/machinekit-developing.asciidoc [hunk 1]
```diff
@@ -86,7 +86,7 @@ As a last step we need to `cd` to a parent directory (making sure the current us
 
 [source,bash]
 ----
-docker run -it --rm -u "$(id -u):$(id -g)" -v "$(pwd):/home/machinekit/build" -w "/home/machinekit/build/machinekit-hal" ${PREFIX}/machinekit-hal-${DISTRUBUTION}-builder-v.${ARCHITECTURE}_${OS_VERSION_NUMBER}:latest debian/buildpackages.py
+docker run -it --rm -u "$(id -u):$(id -g)" -v "$(pwd):/home/machinekit/build" -w "/home/machinekit/build" ${PREFIX}/machinekit-hal-${DISTRUBUTION}-builder-v.${ARCHITECTURE}_${OS_VERSION_NUMBER}:latest /bin/bash -c "debian/buildpackages.py; cp ../*.deb ."
 ----
```
[the remark this answers is not in the document]
Comment (the-snowwhite) [reply docs/developing/machinekit-developing.asciidoc:90]: Well I have tested it to work before commiting so just merge unless you have tested it to have some sort of copy paste error

[the remark this answers is not in the document]
Comment (the-snowwhite) [reply docs/developing/machinekit-developing.asciidoc:90]: as far as I can decode the: `/home/machinekit/build` construction sets up a virtual top folder called build or rather ../build
where the .debs are built. this virtual folder disappears when the docker is exited.
Hence the bash construction for first running the build script and then copying the .debs dow into the repofolder before exiting.

[the remark this answers is not in the document]
Comment (the-snowwhite) [reply docs/developing/machinekit-developing.asciidoc:90]: @cerna 
I have just tested it with building mk-hal arm64 debs...
Safe to merge

**Pull request merged**